<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestButton</class>
 <widget class="QDialog" name="TestButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="normalBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>普通按钮</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkedBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>开关按钮</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menuBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>菜单按钮</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolNormalBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>增强版普通按钮</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolCheckedBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>增强版开关按钮</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolMenuBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>增强版延时菜单按钮</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolNormalActBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>增强版普通按钮</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolArrowBut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>箭头按钮</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolPopMenuBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>增强版即时菜单按钮</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>174</width>
     <height>69</height>
    </rect>
   </property>
   <property name="title">
    <string>单选1</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioCppBut">
      <property name="text">
       <string>C++</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioGoBut">
      <property name="text">
       <string>GO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioPhpBut">
      <property name="text">
       <string>PHP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>90</width>
     <height>129</height>
    </rect>
   </property>
   <property name="title">
    <string>单选2</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioJavaBut">
      <property name="text">
       <string>Java</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioPyBut">
      <property name="text">
       <string>Python</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioRustBut">
      <property name="text">
       <string>Rust</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>73</width>
     <height>129</height>
    </rect>
   </property>
   <property name="title">
    <string>多选</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="cppCheckBox">
      <property name="text">
       <string>C++</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="goCheckBox">
      <property name="text">
       <string>GO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="phpCheckBox">
      <property name="text">
       <string>PHP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="selectLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectBut">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>选中多少</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>100</width>
     <height>177</height>
    </rect>
   </property>
   <property name="title">
    <string>多选-三态设置</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QCheckBox" name="webCheckBox">
      <property name="text">
       <string>WEB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="htmlCheckBox">
         <property name="text">
          <string>HTML</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cssCheckBox">
         <property name="text">
          <string>CSS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="jsCheckBox">
         <property name="text">
          <string>JS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
